README.md

```diff
@@ -84,14 +84,18 @@ A Kubernetes Secret also needs to be defined and mounted to the Pod. The secret
 {
   "AuthSecret": "base64 encoded auth secret",
   "S3SecretKey": "s3 secret key",
-  "S3AccessKey": "s3 access key"
+  "S3AccessKey": "s3 access key",
+  "GithubClientID": "github client ID",
+  "GithubClientSecret": "github client secret"
 }
 ```
 
 `AuthSecret` is the secret for authenticating requests. This is needed by the clients to communicate with kontinuous through JWT.
 
 `S3SecretKey` and `S3AccessKey` are the keys needed to access minio (or S3).
 
+`GithubClientID` and `GithubClientSecret` are optional and only required for Github only authentication. (See below)
```

Comment:
For the deploy tool, can we provide GithubClientID/GithubClientSecret and ask the user for their GH username/password/2FA (so it doesn't get added to the history). That way we can automate the setup process and don't need to do the JWT work?